environment.yml

```diff
@@ -7,3 +7,5 @@ dependencies:
   - networkx
   - pydotplus
   - pytest
+  - sphinx
```

Comment:
if we end up seeing long build times, we can have a test_environment.yml and a docs_environment.yml file separately and use them in the different builds.

Reply:
I've been operating on the assumption that we need most of the dependencies to build the docs, but maybe we don't. This is a good idea. I'll see if I can get it to work.